charts/pulsar/Chart.yaml

```diff
@@ -21,7 +21,7 @@ apiVersion: v1
 appVersion: "2.6.0"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
-version: 2.6.0
+version: 2.6.1
```

Comment:
Since the chart is for pulsar 2.6.0 and we don't release pulsar 2.6.1 yet, I would suggest not bumping the version here. We can use a naming convention like `<pulsar-version>-<minor-version>` for versioning the helm chart. How does that sound?

Reply:
I can do that, the appVersion will match the pulsar version the chart is running. The helm version and pulsar version are different. Do you want them the same so its easy to know what version you are running if you use helm?

Reply:
@sijie Thats what the "appVersion" Value in the Chart.yaml is for. The "version" key is just the version of the helm chart and is completly separate from the actual application version. And when you follow semantic versioning on your helm chart as well then these version will seperate really quickly.

Just a quick look at the Prometheus Helm Chat: Prometheus itself is at 2.20.1 right now but the helm chart is already at 11.x https://github.com/helm/charts/blob/master/stable/prometheus/Chart.yaml#L3-L4

It would be cool to have more frequent updates of this helm chart and not just wait for pulsar updates to release at the same time.

Reply:
I understand the difference between appVersion and version. My recommendation is to add another version. e.g. `2.6.1-0` this means it is a helm chart released for pulsar `2.6.1`.

Reply:
Yes we don't need to align the helm chart release with pulsar release.

Reply:
Not a huge fan of that if when we want to follow the semantic versioning for this helm chart but still better than nothing. Thanks for the new release ;)